lib/liveman/survey/surveys.ex

```diff
@@ -9,16 +9,41 @@ defmodule Liveman.Survey.Surveys do
   def list_surveys do
     survey_list_json = get_json()
 
-    Enum.map(survey_list_json, fn survey_json ->
-      build_survey(survey_json)
-    end)
+    get_survey_list(survey_list_json)
   end
 
   def get_json do
     {_, body} = File.read(@json_file)
     Jason.decode!(body)
   end
 
+  def save_answers(params) do
```

Comment:
Prefer to use, `list_`, `get_`, `create_`, `update_` and `delete_` as prefixes for context methods.

```suggestion
  def create_answer(params) do
```